unfoldingWord/tc-create-app
# solved the issuses 1606, 1607 and 1395

#### Describe what your pull request addresses (Please include relevant issue numbers):
- solved the issues [1606](https://github.com/unfoldingWord/tc-create-app/issues/1606), [1607](https://github.com/unfoldingWord/tc-create-app/issues/1607) and [1395](https://github.com/unfoldingWord/tc-create-app/issues/1395)



## testing 1606(clicking the view column icon):
- Login into tC create app
- Select org 'UW',  Resource 'tN', Lang 'EN' and File 'tit'
- scroll down and click any row (same as we editing the content)
- click on the view column and select the new column from the list (eg: click Reference) (note: scroll a little up to see the changes better )
- It will navigate to the last clicked row and flash the new column on a blue background 

Note: here it won't create a row border instead it flashes the newly added column and also navigates it to the center of the page

## testing 1607(clicking the preview icon):
- Login into tC create app
- Select org 'UW',  Resource 'tN', Lang 'EN' and File 'tit'
- scroll down and click any row (same as we editing the content)
- click on the preview icon (note: scroll a little up to see the changes better )
- It will navigate to the last clicked row and flash the new column on a blue background with a border 

## testing 1395(clicking the page navigation icon):
- Login into tC create app
- Select org 'UW',  Resource 'tN', Lang 'EN' and File 'tit'
- click on the forward navigate icon and it will scroll to the top on the next page
-click on the previous navigate icon and it will scroll to the bottom of the previous page

DOD: Forward page navigation should scroll the user to the top of the page. Backward page navigation should stay scrolled to the bottom.
